### PR TITLE
LG-2487: Segmented control glyph slot

### PR DESCRIPTION
**Pull request by shaneeza**
## ✍️ Proposed changes

🎟 _Jira ticket:_ [LG-2487](https://jira.mongodb.org/browse/LG-2487)
Screen Shot 2022-11-23 at 3 49 34 PM
Screen Shot 2022-11-23 at 3 49 42 PM

## 🛠 Types of changes

- [ ] Tooling (updates to workspace config or internal tooling)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Style change (purely visual updates; if this is the case, attach a screenshot below!)

## ✅ Checklist

- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [x] I have run `yarn changeset` and documented my changes

- [ ] I have added my new package to the global tsconfig
- [ ] I have added my new package to the Table of Contents on the global README

## Feedback Levels

Please try to mark your comments with one of these levels

1. Praise (👏 🚀 🔥 🍾)
2. Opinions, Ideas, Suggestions (💡 👀)
3. Let’s discuss @ Code review (🤷 🤔 🧐 )
4. Doesn’t meet agreed upon standards (😬 ⚠️ ‼️)
5. Missing an edge case (🛑 🙅 ❌)
6. Missing core functionality (⛔️ ☣️ 🚧 🌋 💀)

(Note: Use ⌘-⌃-␣ to open the Emoji keyboard!)

### CODE REVIEW

**Comment by shaneeza** [comment]
todo: update website

File: packages/segmented-control/src/SegmentedControl/styles.ts [hunk 1]
```diff
@@ -13,14 +13,23 @@ export const wrapperStyle = css`
...
+    css`
+      color: ${palette.gray.dark1};
+    `,
+  ),
+  [Theme.Dark]: cx(
+    labelBaseStyles,
+    css`
+      color: ${palette.gray.light1};
+    `,
+  ),
```
Comment (spark33) [inline packages/segmented-control/src/SegmentedControl/styles.ts:32]: Small nit - based on the code styles discussions we've been having, I think `labelBaseStyles` should be passed directly to the `cx` call in the render function than repeatedly defined in the theme styles.

File: packages/segmented-control/README.md [hunk 2]
```diff
@@ -131,8 +131,9 @@ import {
 
 | Prop              | Type                         | Description                                                                                                                                                                 | Default            |
 | ----------------- | ---------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------ |
-| `children`        | `React.ReactNode`            | Can be text and/or an icon element                                                                                                                                          |                    |
+| `children`        | `React.ReactNode`            | Can be text.                                                                                                                                                                |                    |
```
Comment (spark33) [inline packages/segmented-control/README.md:134]: Small nit - maybe this should say 'should' instead of 'can' to express that's what we're expecting

**Review by spark33 — APPROVED** [review]

File: packages/segmented-control/README.md (hunk 2 above)
Comment (shaneeza) [reply packages/segmented-control/README.md:134]: yeah i'll change

**Pull request merged**